### ISSUE TITLE
Need Support For PHP 8.0 

### ISSUE THREAD

**Issue by arunvitto** [Gregwar/CaptchaBundle#222]
@Olaf1989 @Gregwar 
pl. enable PHP 8.0 support in composer update/install 

```
  Problem 1
    - gregwar/captcha-bundle[v2.1.0, ..., v2.1.4] require php ^7.1.3 -> your php version (8.0.2; overridden via config.platform, same as actual) does not satisfy that requirement.

```

**Comment by Gemorroj** [comment]
@Gregwar

**Comment by Gregwar** [comment]
I merged #223 
Thanks for contributing
